chainer/variable.py

```diff
@@ -249,6 +249,28 @@ def zerograd(self):
             else:
                 self._grad.fill(0)
 
+    def scatter_grad(self, array, offset):
```

Comment:
Ditto. Please consider to implement these methods in outside of `Variable` class.